tests/Feature/SnmpTraps/CommonTrapTest.php

```diff
@@ -57,7 +57,7 @@ public function testFindByIp()
 
         Log::shouldReceive('info')->once()->with('Unhandled trap snmptrap', ['device' => $device->hostname, 'oid' => null]);
         Log::shouldReceive('event')->once()->withArgs(function ($e_message, $e_device, $e_type) use ($device) {
-            return $e_message == 'SNMP trap received: ' &&
+            return $e_message == '' &&
                 $device->is($e_device) &&
```

Comment:
```suggestion
            return $e_message == $device->is($e_device) &&
```

Reply:
I thought the best practice was 1 line for all, or one line per subcondition (which is what is implemented now)